MultiSnap_timelapse_ef_230209_3.bsh

```diff
@@ -221,6 +220,10 @@ for(int qy=0;qy<panelnumy;qy++){
 				for(int z=0;z<zstacklength;z++){
 					height = mm.core().getProperty("Focus","Position");
 					Double heightvalue = Double.parseDouble(height);
+					/*mm.core().setProperty("Focus","Position", height + ZinitialShift); //chenge initial z position mi230217 start
```

Comment:
I couldn't understand what this ZinitialShift for. could you tell me later?

Reply:
I would like to disscus in WR. Thank you.